### DISCUSSION TITLE
带有 :props="props" 的el-cascader 选择完成后不自动关闭

### DISCUSSION THREAD

**Discussion by 2512746832** [element-plus/element-plus#6142] [Q&A]
<el-cascader
                style="width: 100%"
                v-model="employee.departmentId"
                :options="department_list"
                :props="props"
                size="small"
              >
</el-cascader>

  const props = {
      expandTrigger: "click",
      children: "children",
      label: "name",
      value:"id",
      checkStrictly: true,
      emitPath:false
    };

打开的级联面板 怎么在勾选一个选择后，自动关闭面板。
现在只能通过点击其他空白区域关闭

**Comment by msidolphin** [comment]
你这是多选吧，多选不会默认关闭的

**Reply by 2512746832** [reply]
不是啊，我这是级联单选， 我没加 multiple: true,

**Comment by 2512746832** [comment]
`<el-cascader
                @change="handleCascaderChange"
                style="width: 100%"
                v-model="employeeAddForm.departmentId"
                :options="department_list"
                :props="props"
                size="small"
              >
const props = {
expandTrigger: "click",
children: "children",
label: "name",
value:"id",
checkStrictly: true,
emitPath:false
};
`

**Comment by 2512746832** [comment]
![QQ截图20220218190655](https://user-images.githubusercontent.com/70892499/154671454-dadd7be1-87a0-4005-a99c-c6bece6758be.png)
![QQ截图20220218190638](https://user-images.githubusercontent.com/70892499/154671471-67392057-6323-49d0-835a-67fa1477a231.png)
![QQ截图20220218190708](https://user-images.githubusercontent.com/70892499/154671486-5b352a89-4384-4ec7-ba1c-8284c49055d0.png)
